### PR TITLE
Add privacy policy link

### PR DESCRIPTION
**Pull request by rafaelji**
Please provide input as to the final link reference.

### CODE REVIEW

**Comment by davrv93** [comment]
Hi @rafaelji  ! thanks for the work :+1: I think that we can use lowercase on href urls,    @m-dotcom can you please provide us the content of the privacy link ? if that could be attached on the trello it would be great ! so maybe @rafaelji  could add it, do you think would it better to have a modal / popup to show that info ? or do you want to go to another page ? I think a modal /popup could help in this scenario to don't lose the focus on the register :smile:    Lord Bless you friends !

**Comment by WarlenOliveira** [comment]
Hi Samuel, I updated the Trello card with the content of the Privacy Policy. I think that we can have a page for that content because in the future we would like to have the pages "FAQ","Terms of Use" and "Privacy Policy" under the "About" option on the Nav Bar. 
About

**Comment by m-dotcom** [comment]
> Hi Samuel, I updated the Trello card with the content of the Privacy Policy. I think that we can have a page for that content because in the future we would like to have the pages "FAQ","Terms of Use" and "Privacy Policy" under the "About" option on the Nav Bar.
> About

We just have to make sure it works well for mobile also.

**Comment by dahngeek** [comment]
Just committed an update of the links so they point to the actual route controllers that show the privacy policy and terms of use pages.

Also included the navbar update @WarlenOliveira was mentioning, in master all links should work (in this branch the page FAQ doesn't exist.)

File: application/views/twigs/auth/register.twig [hunk 1]
```diff
@@ -35,7 +35,8 @@
 		<button class="btn btn-lg btn-block btn-primary" role="button" type="submit">
 			Create account
 		</button>
-		<small>By clicking 'Create Account' you agree to our <a href="/termsOfUse" target="_blank">Terms of Use</a>
+		<small>
+			By clicking 'Create Account' you agree to our <a href="about/terms" target="_blank">Terms of Use</a> and <a href="about/privacy" target="_blank">Privacy Policy.</a>
```
Comment (davrv93) [inline application/views/twigs/auth/register.twig]: It's ok, it works

File: application/views/twigs/template.twig [hunk 2]
```diff
@@ -51,13 +51,20 @@
 				<div class="collapse navbar-collapse justify-content-between" id="navbar-collapse">
 					<ul class="navbar-nav">
 						<li class="nav-item mr-2">
-							<a class="nav-link" href="/products">Available Products</a>
+							<a class="nav-link" href="/products">Products</a>
```
Comment (davrv93) [inline application/views/twigs/template.twig]: Could you please make a new PR for this ? :)

File: application/views/twigs/template.twig [hunk 3]
```diff
@@ -51,13 +51,20 @@
...
 						</li>
 						<li class="nav-item mr-2">
-							<a class="nav-link" href="/projects">Ongoing Translations</a>
+							<a class="nav-link" href="/projects">Translations</a>
```
Comment (davrv93) [inline application/views/twigs/template.twig]: The same comment, please include it in a new PR related to the trello changes

File: application/views/twigs/template.twig [hunk 4]
```diff
@@ -51,13 +51,20 @@
...
-							<a class="nav-link" href="/about/faq">FAQ</a>
+						<li class="nav-item dropdown mr-2">
+							<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+							About
+							</a>
+							<div class="dropdown-menu" aria-labelledby="navbarDropdown">
+							<a class="dropdown-item" href="/about/faq">FAQ</a>
+							<a class="dropdown-item" href="/about/privacy">Privacy policy</a>
+							<a class="dropdown-item" href="/about/terms">Terms of use</a>
+							</div>
```
Comment (davrv93) [inline application/views/twigs/template.twig]: The same comment, please include it in a new PR related to the trello changes

File: application/views/twigs/template.twig (hunk 2 above)
Comment (dahngeek) [reply application/views/twigs/template.twig]: done

File: application/views/twigs/template.twig [hunk 5]
```diff
@@ -51,20 +51,10 @@
...
-							<a class="nav-link" href="/projects">Translations</a>
-						</li>
-						<li class="nav-item dropdown mr-2">
-							<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-							About
-							</a>
-							<div class="dropdown-menu" aria-labelledby="navbarDropdown">
-							<a class="dropdown-item" href="/about/faq">FAQ</a>
-							<a class="dropdown-item" href="/about/privacy">Privacy policy</a>
-							<a class="dropdown-item" href="/about/terms">Terms of use</a>
```
Comment (davrv93) [inline application/views/twigs/template.twig]: :+1:

**Pull request merged**